sdk/automanage/Azure.ResourceManager.Automanage/src/autorest.md

```diff
@@ -0,0 +1,177 @@
+# Generated code configuration
+Run `dotnet build /t:GenerateCode` to generate code.
+``` yaml
+save-inputs: true
+azure-arm: true
+csharp: true
+library-name: automanage
+namespace: Azure.ResourceManager.automanage
```

Comment:
Upper case the first character of `automanage`

Reply:
Done